src/components/Postiterie.vue

```diff
@@ -1,203 +1,454 @@
+    this.handleClientLoad();
   }
 }
 </script>
 
+<style scoped>
+  span{
+      font-weight: 100;
+      width: 20%;
+      text-align: center;
+  }
 
-<style>
-.rect {
-    width: 80px;
-    height: 80px;
-    position: absolute;
-    left: -1000px;
-    top: -1000px;
-}
-</style>
+  .button-container{
+      width: 100%;
+      margin-top: 1em;
+      display: flex;
+      flex-direction: column;
+      align-items:center;
+      -webkit-touch-callout: none; /* iOS Safari */
```

Comment:
Can you load a vue plugin (or webpack, whatever) to handle all browsers for you?

Reply:
Good news from VueCLI's doc, Postcss seems to be used with autoprefixer enabled by default. It even almost runs on IE